canvasapi/outcome.py

```diff
@@ -0,0 +1,319 @@
+            or `GET /api/v1/courses/:course_id/outcome_groups/:id/subgroups \
+            <https://canvas.instructure.com/doc/api/outcome_groups.html#method.outcome_groups_api.subgroups`_
+
+        :returns: Paginated List of OutcomeGroups linked to the current group.
+        :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
+            :class:`canvasapi.outcome.OutcomeGroup`
+        """
+        return PaginatedList(
+            OutcomeGroup,
+            self._requester,
+            'GET',
+            '%s/outcome_groups/%s/subgroups' % (self.context_ref(), self.id)
+        )
+
+    def create_subgroup(self, title, **kwargs):
+        """
+        Create a subgroup of the current group
+
+        :calls: `POST /api/v1/global/outcome_groups/:id/subgroups \
+            <https://canvas.instructure.com/doc/api/outcome_groups.html#method.outcome_groups_api.create>`_
+            or `POST /api/v1/accounts/:account_id/outcome_groups/:id/subgroups \
+            <https://canvas.instructure.com/doc/api/outcome_groups.html#method.outcome_groups_api.create>`_
+            or `POST /api/v1/courses/:course_id/outcome_groups/:id/subgroups \
+            <https://canvas.instructure.com/doc/api/outcome_groups.html#method.outcome_groups_api.create>`_
+
+        :param title: The title of the subgroup.
+        :type title: str
+
+        :returns: Itself as an OutcomeGroup object.
+        :rtype: :class:`canvasapi.outcome.OutcomeGroup`
+        """
+        response = self._requester.request(
+            'POST',
+            '%s/outcome_groups/%s/subgroups' % (self.context_ref(), self.id),
+            title=title,
+            _kwargs=combine_kwargs(**kwargs)
+        )
+
+        return OutcomeGroup(self._requester, response.json())
+
+    def import_outcome_group(self, source_outcome_group_id):
+        """
+        Import an outcome group as a subgroup into the current outcome group
+
+        :calls: `POST /api/v1/global/outcome_groups/:id/import \
+            <https://canvas.instructure.com/doc/api/outcome_groups.html#method.outcome_groups_api.import>`_
+            or `POST /api/v1/accounts/:account_id/outcome_groups/:id/import \
+            <https://canvas.instructure.com/doc/api/outcome_groups.html#method.outcome_groups_api.import>`_
+            or `POST /api/v1/courses/:course_id/outcome_groups/:id/import \
+            <https://canvas.instructure.com/doc/api/outcome_groups.html#method.outcome_groups_api.import>`_
+
+        :param source_outcome_group_id: The id of the Outcome Group to be imported.
+        :type source_outcome_group_id: int
+
+        :returns: Itself as an OutcomeGroup object.
+        :rtype: :class:`canvasapi.outcome.OutcomeGroup`
+        """
+        response = self._requester.request(
+            'POST',
+            '%s/outcome_groups/%s/import' % (self.context_ref(), self.id),
+            source_outcome_group_id=source_outcome_group_id
```

Comment:
(optional)
we might be able to leverage obj_or_id for the `source_outcome_group_id` param?